pkg/cloud/services/networking/service.go

```diff
@@ -78,3 +83,23 @@ func NewTestService(projectID string, client NetworkClient, logger logr.Logger)
 		logger:    logger,
 	}
 }
+
+// replaceAllAttributesTags replaces all tags on a neworking resource.
+// the value of resourceType must match one of the allowed constants: trunkResource or portResource.
+func (s *Service) replaceAllAttributesTags(eventObject runtime.Object, resourceType string, resourceID string, tags []string) error {
+	if resourceType != trunkResource && resourceType != portResource {
+		record.Warnf(eventObject, "FailedReplaceAllAttributesTags", "Invalid resourceType argument in function call")
```

Comment:
nit: Warnf is redundant. The only reason we're here is because somebody's hacking on it and they messed up.